handler.go

```diff
@@ -119,44 +119,51 @@ func MakeHandlerWithBody(app *App, fn HandlerWithBodyFunc, spec Spec) http.Handl
 		} else {
 			status = http.StatusOK
 		}
+		marshaled, err := json.Marshal(response)
+		if err != nil {
+			status = http.StatusInternalServerError
+			marshaled, _ = json.Marshal(Response{
+				"message": fmt.Sprintf("Failed to marshal response: %v %v", response, err)})
+		}
 
-		log.Infof("Request(Status=%v, Method=%v, Path=%v, Form=%v, Time=%v, Error=%v)",
-			status, r.Method, r.URL, r.Form, elapsedTime, err)
+		log.Infof("%s %s %s %s", r.Method, r.URL, status, marshaled)
```

Comment:
Why did this need to change? These are my issues with the new format:

* Status code will be lost between URL and marshaled response. Previously it was the first field and it was easy to visually scan logs for errors.
* Request time is no longer logged.
* Real error is no longer logged (unless I'm missing something).
* Form is no longer logged which is *very* useful for POST requests.